arm/ripper/main.py

```diff
@@ -375,16 +377,21 @@ def main(logfile, job):
     args = entry()
 
     devpath = "/dev/" + args.devpath
```

Comment:
Removing the "/dev/" + from here would allow you to push the device( ${DEVNAME} ) in from docker_arm_wrapper.sh. Allowing for multiple drives within a single container.